src/lib/flexbox/api/flex-offset.ts

```diff
@@ -84,7 +103,43 @@ export class FlexOffsetDirective extends BaseFxDirective implements OnInit, OnCh
   // Protected methods
   // *********************************************
 
+  /** The flex-direction of this element's host container. Defaults to 'row'. */
+  protected _layout = 'row';
```

Comment:
Why? I thought explicit value assignment defines type and negates need to declare it.